projects/admin/src/app/circulation/patron/card/card.component.html

```diff
@@ -1,18 +1,18 @@
 <!--
  RERO ILS UI
- Copyright (C) 2019 RERO
-
- This program is free software: you can redistribute it and/or modify
- it under the terms of the GNU Affero General Public License as published by
- the Free Software Foundation, version 3 of the License.
-
- This program is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- GNU Affero General Public License for more details.
-
- You should have received a copy of the GNU Affero General Public License
- along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ Copyright (C) 2019 RERO
```

Comment:
--> 2021

Reply:
I just remove a strange character in this header comment. Not creating the component ;-)